ckanext/stcndm/schemas/presets.yaml

```diff
@@ -249,3 +249,8 @@ presets:
   values:
     form_snippet: codeset_select.html
     display_snippet: select.html
+
+- preset_name: codeset_multiple_select
+  values:
+    form_snippet: codeset_multiple_select.html
+    display_snippet: multiple_choice.html
```

Comment:
You probably need the validators like in multiple_select, don't you? https://github.com/open-data/ckanext-scheming/blob/master/ckanext/scheming/presets.json#L81-L87